scripts/generator/graphql_generator/csharp/SDK/GlobalGameObject.cs.erb

```diff
@@ -0,0 +1,17 @@
+#if !SHOPIFY_MONO_UNIT_TEST
+namespace <%= namespace %>.SDK {
+    using UnityEngine;
+
+    public class GlobalGameObject {
+        private static GameObject GameObject;
+
+        public static T AddComponent<T>() where T: Component {
+            if (GameObject == null) {
+                GameObject = new GameObject("Shopify");
+            }
+
+            return GameObject.AddComponent<T>();
```

Comment:
I feel like `GlobalGameObject` should not. You should be able to add multiple components at the same time.

For `UnityLoader` for instance lets say you're doing multiple loads at the same time you need to add multiple components. Although I guess we could refactor `UnityLoader` to not destroy itself but you can have multiple loads.

For `UnitySetTimeout` in #78 we check if a component already exists and if it does not just simply use that existing component.

I feel like for this PR `GlobalGameObject` would not check for the existence of components but if we see it being valuable then we can add it in.